.github/workflows/reusable-azure-ai-translate.yml

```diff
@@ -76,7 +74,7 @@ jobs:
           path: 'translation-action-scripts'
           sparse-checkout: '.github/scripts/translate.py'
           sparse-checkout-cone-mode: false
-          persist-credentials: false
```

Comment:
This one should be able to remain `false`. We're not committing or pushing any code to the `workflows` repository, just using the script. The calling repository in the first checkout is the one we're concerned with here.